### ISSUE TITLE
Markdown field adding strange content and overflowing

### ISSUE THREAD

**Issue by jameswoodley** [OrchardCMS/OrchardCore#11388]
### Describe the bug
Using `OrchardCore.Application.Cms.Core.Targets` v1.3.0 we are seeing strange behaviour with the Markdown field. Upon loading the Part it shows a collection of x:
![image](https://user-images.githubusercontent.com/52610563/158419483-718fa37c-350b-43db-8d26-c1fea4c0bdf0.png)

This also happens if creating a new instance of the Content Type, the Markdown field is prefilled with the xxxxxxx until clicking on it

Clicking on the field does then show the content, however it seems to overflow it's container
![image](https://user-images.githubusercontent.com/52610563/158419597-4b83a1de-7f18-4bb0-adae-6faf40f401c7.png)

This doesn't seem to be the case on the Try Orchard site [https://try.orchardcore.net/qj4dh3wy/](https://try.orchardcore.net/qj4dh3wy/)
![image](https://user-images.githubusercontent.com/52610563/158419804-4ed773d8-baf1-419b-94cf-3fc327c3708d.png)

### To Reproduce
1. Open Content Type with markdown field
2. Click on 'the field
3. See content, however overflowing it's container

### Expected behavior
As it currently works on the Try Orchard site - makes me wonder if this is an upgrade issue as we recently upgraded to v1.3.0

This could be related to #10495 ?

**Comment by dsghi** [comment]
I have a couple of issues with the Markdown field as well...  hopefully, it's OK to pile on this issue...

Clicking the Toggle Preview toolbar button takes it full screen, but it ends up behind the top and left navigation. (If you first click side by side or full screen, you can switch to preview without issue):
![](https://content.screencast.com/users/DerekGabriel/folders/Snagit/media/58d9b3f3-2b4c-41c7-9336-fc8a54c4ee97/05.05.2022-18.01.png)

![](https://content.screencast.com/users/DerekGabriel/folders/Snagit/media/08d7de2c-5c39-4d33-8cbd-50173032b85e/05.05.2022-18.02.png)

If using the admin dark mode, the content of the markdown field is not visible:
![](https://content.screencast.com/users/DerekGabriel/folders/Snagit/media/3b148a76-3b06-49c6-a4e0-5fc497a37838/05.05.2022-18.04.png)

**Comment by dsghi** [comment]
Well, it turns out this is a super easy fix to all of the above, lol - it's an issue with loading the codemirror.css - **a quick workaround is to disable Use framework CDN from Settings > General > Resources tab**.

Found this in the browser console: Failed to find a valid digest in the 'integrity' attribute for resource 'https://cdnjs.cloudflare.com/ajax/libs/codemirror/5.65.2/codemirror.css' with computed SHA-384 integrity 'bsaAhvdduZPAwUb7RRLRvDgtEtOsggrgjkr/EjPO1i/vdoi+DmdLaG79UOt6M5hD'. The resource has been blocked.

Seems the SHA has changed on the CDN, when I looked they're publishing a 512 now by default, IDK if the 384 is available too, I can't seem to locate it. I've read that 384 is the most secure... but anyway.

I see this is being addressed with #11646
